src/Evolution/BoundaryCorrectionTags.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "NumericalAlgorithms/SpatialDiscretization/OptionTags.hpp"
+
+namespace evolution {
+namespace OptionTags {
+/// The boundary correction used for coupling together neighboring cells or
+/// applying boundary conditions.
```

Comment:
Most people reading a DG textbook will be looking for something called "numerical flux". I think it'd be useful to use those words in this documentation. Perhaps by saying "For systems blah, this correction is also called the numerical flux" or similar?

Make same change to documentation of tag below.